router.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+)
+
+var NoBody = ""
+
+type MyMux struct {}
+
+func (p *MyMux) ServeHTTP(w *Response, r *Request) string {
```

Comment:
you should have a test for each of these tests.

Reply:
✅